conveyor.conf

```
@@ -55,9 +58,11 @@ app {
   }
 
   windows {
+    inputs += cdr-client-service/conf/default-application-customer.yaml -> conf/default-application-customer.yaml
```

doesn't the 'default' need to be gone? (as it is also referenced as such on line 183)
```suggestion
    inputs += cdr-client-service/conf/default-application-customer.yaml -> conf/application-customer.yaml
```